app/models/contact.rb

```diff
@@ -0,0 +1,3 @@
+class Contact < ApplicationRecord
+  belongs_to :users
```

Comment:
đọc lại association em nhé
https://guides.rubyonrails.org/association_basics.html#the-types-of-associations